rest/src/routes/routeUtils.js

```diff
@@ -128,10 +131,15 @@ const routeUtils = {
 	 * @returns {object} Parsed paging options.
 	 */
 	parsePagingArguments: args => {
-		const parsedOptions = { id: undefined, pageSize: 0 };
```

Comment:
reverted to old function, those changes make no sense anymore here (probably forgot to delete) - need to change related tests